tests/podcast-grid-layout.spec.ts

```diff
@@ -0,0 +1,185 @@
+    const testingFilterLink = page.getByRole('list', { name: 'topics' }).getByRole('link', { name: 'testing' });
+    await expect(testingFilterLink).toBeVisible();
+
+    // Check "All Episodes" section
+    await expect(page.getByRole('heading', { level: 2, name: 'All Episodes' })).toBeVisible();
+    await expect(page.getByText('28 episodes')).toBeVisible();
+
+    // Check grid layout - should have multiple podcast cards
+    const podcastCards = page.getByRole('article').filter({ hasNot: page.locator('iframe') });
+    await expect(podcastCards).toHaveCount(28); // Should match total episodes displayed
+
+    // Check that podcast cards have proper structure
+    const firstCard = podcastCards.first();
+    const cardImage = firstCard.locator('img').first();
+    await expect(cardImage).toBeVisible(); // Podcast cover image
+    await expect(firstCard.getByRole('heading', { level: 2 })).toBeVisible(); // Podcast title
+    const cardLink = firstCard.getByRole('link').first();
+    await expect(cardLink).toBeVisible(); // Clickable links
+
+    // Check call-to-action section
+    await expect(page.getByRole('heading', { level: 3, name: 'Want to collaborate?' })).toBeVisible();
+    await expect(page.getByText('I\'m always open to interesting podcast conversations')).toBeVisible();
+    
+    // Check LinkedIn link
+    const linkedInLink = page.getByRole('link', { name: 'Get in touch' });
+    await expect(linkedInLink).toBeVisible();
+    await expect(linkedInLink).toHaveAttribute('href', 'https://www.linkedin.com/in/debbie-o-brien-1a199975/');
+    await expect(linkedInLink).toHaveAttribute('target', '_blank');
+  });
+
+  test('podcast card interactions and hover effects', async ({ page }) => {
+    await page.goto('/podcasts');
+
+    // Get first podcast card (excluding featured podcast with iframe)
+    const podcastCards = page.getByRole('article').filter({ hasNot: page.locator('iframe') });
+    const firstCard = podcastCards.first();
+
+    // Test that card contains expected elements
+    const cardImage = firstCard.locator('img').first();
+    await expect(cardImage).toBeVisible();
+    await expect(firstCard.getByRole('heading', { level: 2 })).toBeVisible();
+    await expect(firstCard.locator('time')).toBeVisible(); // Date element
+    
+    // Test that links are clickable
+    const cardLink = firstCard.getByRole('link').first();
+    await expect(cardLink).toBeVisible();
+    
+    // Test that tags are present
+    const tagLinks = firstCard.getByRole('link').filter({ hasText: /^(testing|playwright|nuxt|react|bit|dev rel|jamstack|mentoring)$/ });
+    await expect(tagLinks.first()).toBeVisible();
+  });
+
+  test('LinkedIn button functionality', async ({ page, context }) => {
+    await page.goto('/podcasts');
+
+    // Click the "Get in touch" button
+    const [linkedInPage] = await Promise.all([
+      context.waitForEvent('page'),
+      page.getByRole('link', { name: 'Get in touch' }).click()
```

Comment:
instead of actually clicking this link and checking it goes to a third party page we should mock it